src/Athena/Views/Home/Schedule_Page.cshtml

```diff
@@ -0,0 +1,32 @@
+﻿
+@{
+    ViewData["Title"] = "Schedule_Page";
+}
+
+<h2>Schedule</h2>
+
+
+
+
+<div id="name"><ul></ul></div>
+
+
+
+
+<div id="first">
+    <h1 id="specific" class="btn" style="width:30em;background-color:navy">Degree Specific Courses</h1>
+    <div>
+        <p>First Section Content</p>
+        <input id="enter" type="text">
+        <input type="button" value="Enter name" onclick="displayUserName()">
+    </div>
+    <br />
+    <br />
+    <h1 class="btn" style="width:30em;background-color:navy">All Courses</h1>
```

Comment:
Use the [material color palette](http://materializecss.com/color.html) for color. This will also help eliminate inline styles.